src/shogun/distributions/Gaussian.cpp

```diff
@@ -267,7 +267,7 @@ float64_t CGaussian::compute_log_PDF(SGVector<float64_t> point)
 	return -0.5 * answer;
 }
 
-SGVector<float64_t> CGaussian::get_mean()
+SGVector<float64_t>& CGaussian::get_mean()
```

Comment:
mmm, but I got error message like 
error: no matching function for call to 'add', if I don't return reference. More error message in here https://pastebin.mozilla.org/9025837